device-types/Cisco/MDS-C9148S.yaml

```diff
@@ -0,0 +1,120 @@
+---
+manufacturer: Cisco
+model: MDS 9148S
+slug: ds-c9148s
+part_number: DS-C9148S
+u_height: 1
+is_full_depth: true
+power-ports:
```

Comment:
It looks like these power supplies are modular. Is that true? If so, then these should be inside of `module-bays` and the associated modular PSUs should be added to the module-types folder if you can